ulib/experimental/Steel.SLDump.fsti

```diff
@@ -0,0 +1,23 @@
+module Steel.SLDump
+
+include Steel.SLDump.Base
+include Steel.Effect
+include Steel.Effect.Atomic
+
+val sldump'
```

Comment:
Why do you need `sldump'` and `sldump`?

Reply:
I provide a version without any `framing_implicit` arguments for those who want to explicitly instantiate implicit arguments. I have some trouble calling functions with `framing_implicit` arguments when explicitly instantiating some, but not all such implicits.

Reply:
What are the issues that you observe? I'm a bit surprised that there is a difference in behavior between a tagged implicit provided explicitly, and the same argument marked explicit, I can't remember observing this when debugging the Steel tactic.